### PR TITLE
change problem on portfolio_view and add navbar to template_view

### PR DESCRIPTION
**Pull request by Tokarchyk**
change some code

add new elements to template_view

### CODE REVIEW

File: application/views/portfolio_view.php [hunk 1]
```diff
@@ -1,51 +1,29 @@
...
-    <input type="hidden" id="email" name="email" value="<?php echo $_SESSION['user_email'] ?>" placeholder="yourEmail@gmail.com">
-
-    <?php else: ?>
-        <input type="email" id="email" name="email" value="" placeholder="yourEmail@gmail.com">
-        <?php endif; ?>
-
+    <?php if (array_key_exists('user_email', $_SESSION) && !empty($_SESSION['user_email'])){ ?>
+        <label for="email">Email:</label>
+    <input type="hidden" name="user_email" value="<?php echo $_SESSION['user_email']; ?>" />
+<?php }; ?>
```
Comment (S-Husach) [inline application/views/portfolio_view.php]: Indent, psr

File: application/views/portfolio_view.php [hunk 2]
```diff
@@ -1,51 +1,29 @@
...
-    ?>
-
-    <label for="email">Email:</label>
-    <input type="hidden" id="email" name="email" value="<?php echo $_SESSION['user_email'] ?>" placeholder="yourEmail@gmail.com">
-
-    <?php else: ?>
-        <input type="email" id="email" name="email" value="" placeholder="yourEmail@gmail.com">
-        <?php endif; ?>
-
+    <?php if (array_key_exists('user_email', $_SESSION) && !empty($_SESSION['user_email'])){ ?>
```
Comment (S-Husach) [inline application/views/portfolio_view.php]: psr

File: application/views/template_view.php [hunk 3]
```diff
@@ -9,7 +9,21 @@
...
+	</form>
 		<?php include 'application/views/'.$content_view; ?>
 	</body>
 </html>
+
+<?php
+if (isset($_POST['logout'])) {
+	unset($_SESSION['user_email']);
+	session_destroy();
+}
```
Comment (S-Husach) [inline application/views/template_view.php]: Why do we place logic in the view file? It should be in controller

**Review by S-Husach — CHANGES_REQUESTED** [review]

File: application/views/template_view.php (hunk 3 above)
Comment (Tokarchyk) [reply application/views/template_view.php]: send this logic to AuthorizationController

File: application/views/portfolio_view.php (hunk 2 above)
Comment (Tokarchyk) [reply application/views/portfolio_view.php]: update

File: application/views/portfolio_view.php (hunk 1 above)
Comment (Tokarchyk) [reply application/views/portfolio_view.php]: removed duplicate input. left one condition.
        if (array_key_exists('user_email', $_SESSION) && !empty($_SESSION['user_email'])){ ?>
        <label for="email">Email:</label>
        <input type="hidden" name="user_email" value="<?php echo $_SESSION['user_email']; ?>" />

File: application/views/portfolio_view.php [hunk 4]
```diff
@@ -8,10 +8,11 @@
 ?>
 <form onsubmit="event.preventDefault();" 
     action="/comment/store" method="POST">
-    <?php if (array_key_exists('user_email', $_SESSION) && !empty($_SESSION['user_email'])){ ?>
+    <?php 
+        if (array_key_exists('user_email', $_SESSION) && !empty($_SESSION['user_email'])){ ?>
```
Comment (S-Husach) [inline application/views/portfolio_view.php]: Still can't see space between ) and {

File: application/views/template_view.php [hunk 5]
```diff
@@ -9,21 +9,95 @@
...
-	session_destroy();
+
+
+
+
+
+
+
+
+
```
Comment (S-Husach) [inline application/views/template_view.php]: What is this for?

File: application/views/template_view.php [hunk 6]
```diff
@@ -9,21 +9,95 @@
...
+  span.psw {
+    display: block;
+    float: none;
+  }
+  .cancelbtn {
+    width: 100%;
+  }
 }
-?>
+</style>
```
Comment (S-Husach) [inline application/views/template_view.php]: Please add empty line at the end

**Review by S-Husach — CHANGES_REQUESTED** [review]

File: application/views/portfolio_view.php (hunk 4 above)
Comment (Tokarchyk) [reply application/views/portfolio_view.php]: added

Comment (S-Husach) [reply application/views/portfolio_view.php]: no

File: application/views/portfolio_view.php [hunk 7]
```diff
@@ -132,27 +111,21 @@ function hideEdit(id) {
...
-    
-    if ($_SESSION['user_email'] === $row['email']){
-        echo '<button class="btn-del" style="display:block" onclick="deleteComment(' . $row['id'] . ')" type="button">Delete</button>'; 
-        echo '<button class="showEdit" style="display:block" onclick="showEdit(' . $row['id'] . ')">Edit</button>';
-        echo '<div style="display:none" class="div-update-comment-">';
-        echo '<input  type="text" class="update-comment-" value="' . htmlspecialchars($row['comment']) . '">';
-        echo '<button class="btn-save" onclick="updateComment(' . $row['id'] . ')" type="button" >Save</button>';
-        echo '<button class="btn-cancel" onclick="hideEdit(' . $row['id'] . ')">Cancel</button>';
-        echo '</div>';
+        if (array_key_exists('user_email', $_SESSION) && !empty($_SESSION['user_email'])){
```
Comment (S-Husach) [inline application/views/portfolio_view.php]: same

Comment (S-Husach) [reply application/views/portfolio_view.php]: and looks like some extra spaces here

File: application/views/template_view.php (hunk 5 above)
Comment (Tokarchyk) [reply application/views/template_view.php]: I added css for button. removed these lines

File: application/views/template_view.php [hunk 8]
```diff
@@ -9,7 +9,95 @@
 	<script src="https://code.jquery.com/jquery-3.7.1.min.js" integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous"></script>	
 	</head>
 	<body>
-
+  		<button type ="submit"><a href="/comment">Comment</a></button>
+  		<button type ="submit"><a href="/authorization">Sing in</a></button>
+  		<button type ="submit"><a href="/registration">Create account</a></button>
+  	<form action="/authorization/logout" method="POST">
+		<button name="logout_btn" type="submit" value="logout">Sing out</button>
+	</form>
```
Comment (S-Husach) [inline application/views/template_view.php]: why different indents?

File: application/controllers/AuthorizationController.php [hunk 9]
```diff
@@ -76,22 +71,13 @@ public function action_hashing()
...
-        
-        // var_dump($password1); 
-
-        // if(password_verify($password, $password1)){
-        //     var_dump('true');
-        // } else {
-        //     var_dump('false');
-        // };
+    }
+     public function action_logout()
```
Comment (S-Husach) [inline application/controllers/AuthorizationController.php]: extra space before public
no empty line between methods

**Review by S-Husach — CHANGES_REQUESTED** [review]

File: application/views/template_view.php (hunk 6 above)
Comment (Tokarchyk) [reply application/views/template_view.php]: done

**Review by S-Husach — APPROVED** [review]

**Pull request merged**